app/views/directives/logs/_log-viewer.html

```diff
@@ -9,9 +9,8 @@
     <span ng-if="kibanaAuthUrl">
       <form
         action="{{kibanaAuthUrl}}"
-        method="POST">
+        method="GET">
```

Comment:
This should no longer be a form. It can be something like

```html
<a ng-href="{{kibanaAuthUrl}}>View Archive</a>
```

We should remove "auth" from the name if it no longer handles auth.

Reply:
Oh this should be using `kibanaArchiveURL` and we can remove `kibanaAuthURL`. I see now why you need to prefix the URL.. It used to be passed as a redirect.